lib/sitemap/sitemap_generator.rb

```diff
@@ -11,17 +11,36 @@ def self.sitemap_limit
   end
 
   def get_all_documents
+    search_config = SearchConfig.instance
+    search_body = {
+      query: {
+        bool: {
```

Comment:
I think this bit could also go in the filter context, since you don't care about scoring.

https://www.elastic.co/guide/en/elasticsearch/reference/2.4/query-filter-context.html